layer/filestore_unix.go

```diff
@@ -0,0 +1,13 @@
+// +build !windows
```

Comment:
Obviously needed for compilation. I didn't want to have separate interfaces between Unix and Windows - and besides, in the future, you may want to support Windows containers on Linux ;).  The alternate is to actually do the write on Unix, but it's an overhead which isn't strictly needed.